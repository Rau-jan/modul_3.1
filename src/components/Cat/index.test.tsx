import { Cat } from '.';
import { expect, describe, it } from 'vitest';
import { render } from '@testing-library/react';

describe('Alert', () => {
  it('show component', () => {
    const { container } = render(<Cat />);
    expect(container).matchSnapshot();
  });
});
