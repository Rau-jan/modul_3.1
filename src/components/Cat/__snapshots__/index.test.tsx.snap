// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Alert > show component 1`] = `
<div>
  <div />
</div>
`;
